backend/src/main/java/com/empire/World.java

```diff
@@ -65,7 +65,10 @@
 	boolean gameover;
 
 	private static Gson getGson() {
-		return new GsonBuilder().setFieldNamingPolicy(FieldNamingPolicy.LOWER_CASE_WITH_UNDERSCORES).create();
+		return new GsonBuilder()
+				.enableComplexMapKeySerialization()
```

Comment:
I took a look at the docs on this, is my assumption correct that you want to see kingdoms (NationData objs) not keyed by their somewhat arbitrary names (if so, I agree this is proper schema design)?

Reply:
This is basically an ugly edge case in Gson: gson by default serializes all map keys using .toString(), even if they have an @SerializedName annotation. Consequently, we fail to deserialize such maps. This broke when I introduced the gothi map, but didn't notice it until now.